core/app/models/spree/taxon.rb

```diff
@@ -26,7 +26,7 @@ class Taxon < Spree::Base
     after_touch :touch_ancestors_and_taxonomy
 
     has_attached_file :icon,
-      styles: { mini: '32x32>', normal: '128x128>' },
+      styles: { mini: '32x32>', normal: '128x128>', medium: '400x400>', large: '600x600>' },
```

Comment:
Not all stores need all these sizes. This is a rarely used feature, but if a store is using this we force them to pre-process four sizes now, what is not what we want to force stores into. 

Could you make this change in your particular store instead? Here is an article on how to achieve this https://github.com/solidusio/solidus/blob/master/guides/products-and-variants/product-images.md#paperclip-settings (Just replace `Spree::Image` with `Spree::Taxon`)